smartgraph.css

```diff
@@ -35,26 +36,41 @@
 /* Custom vertex. You should revert any unwanted styling in the default element,
    style, since custom styles will be appended to the default style */
 .traitor {
-    -fx-stroke-width: 3;
-    -fx-stroke: red;
-    -fx-stroke-type: inside;
     -fx-fill: #f28888;
 }
 
+.loyal {
+    -fx-fill: #B1DFF7;
+}
+
 .attack {
-    -fx-stroke-width: 5;
-    -fx-stroke: green;
-    -fx-stroke-type: inside; /* you should keep this for vertex.radius to hold */
+    -fx-stroke: #cb0000;
 }
 
 .defense {
-    -fx-stroke-width: 5;
-    -fx-stroke: purple;
-    -fx-stroke-type: inside; /* you should keep this for vertex.radius to hold */
+    -fx-stroke: #000000;
+}
+
+.none {
+    -fx-effect: none;
+}
+
+.commander {
+    -fx-effect: dropshadow(three-pass-box, orange, 10, 0, 0, 0);
```

Comment:
Please consider making wider shadow, because on smaller displays it is hardly visible.

Reply:
I made the shadow slighty wider and changed the colors. Is it more visible now?